README.md

```diff
@@ -87,6 +92,15 @@ skupper-service-controller-57cdbb56c5-vc7s2   1/1     Running   0          34s
 skupper-site-controller-d7b57964-gxms6        1/1     Running   0          51m
 ```
 
+You can now navigate to the Skupper console.
+
+```
+$ kubectl get routes
+```
+
+Navigate to the `skupper` route and use `admin/admin` to log in.
+
+
```

Comment:
One comment only would be to mention that this is for OpenShift only.
Maybe we could also explain how to get to the console on Minikube as well.

Reply:
@fgiorgetti  what are you suggesting? using nodeport? are there changes to the yaml required?

Reply:
Instead of using `kubectl get routes`, we should use: `skupper -n <namespace> status` which returns the console URL,
independently of whether you are running against a minikube or OpenShift cluster.